tests/e2e/Services/Databases/DatabasesBase.php

```diff
@@ -983,6 +1002,100 @@ public function testCreateDocument(array $data): array
+                    Permission::update(Role::user($this->getUser()['$id'])),
+                    Permission::delete(Role::user($this->getUser()['$id'])),
+                ]
+            ]);
+        }
+
+        $docs = [];
+        for ($i = 0; $i <= 2; $i++) {
+            $docs[] = $this->client->call(Client::METHOD_GET, '/databases/' . $data['databaseId'] . '/collections/' . $data['moviesId'] . '/documents', array_merge([
+                'content-type' => 'application/json',
+                'x-appwrite-project' => $this->getProject()['$id'],
+                'x-appwrite-timeout' => 1,
+            ], $this->getHeaders()), [
+                //'queries' => ['sleep("$id", 1)'],
+                'queries' => ['notEqual("longtext", "appwrite")'],
+            ]);
+        }
+
+        for ($i = 0; $i < count($documents); $i++) {
+            $this->client->call(Client::METHOD_DELETE, '/databases/' . $data['databaseId'] . '/collections/' . $data['moviesId'] . '/documents/' . $documents[$i]['body']['$id'], array_merge([
+                'content-type' => 'application/json',
+                'x-appwrite-project' => $this->getProject()['$id'],
+            ], $this->getHeaders()));
+        }
+
+        $this->assertEquals(408, $docs[0]['headers']['status-code']); // insert
+        $this->assertEquals(403, $docs[1]['headers']['status-code']); // update
+        $this->assertEquals(403, $docs[2]['headers']['status-code']); // blocked
+    }
+
+    /**
+     * @depends testCreateDocument
+     */
+    public function testConsoleTimeouts(array $data): void
+    {
+        $documents = $this->client->call(Client::METHOD_GET, '/databases/' . $data['databaseId'] . '/slow-queries', array_merge([
+            'content-type' => 'application/json',
+            'x-appwrite-project' => $this->getProject()['$id'],
+        ], $this->getHeaders()), [
+            'queries' => [
+                'equal("blocked", true)',
+                'equal("collectionId", "' . $data['moviesId'] . '")',
+                'orderAsc("$updatedAt")'
+            ]
+        ]);
+
+        $this->assertEquals(200, $documents['headers']['status-code']);
+        $this->assertEquals(1, $documents['body']['total']);
+        $this->assertEquals(true, $documents['body']['documents'][0]['blocked']);
+        $this->assertEquals(2, $documents['body']['documents'][0]['count']);
+
+        $document = $this->client->call(Client::METHOD_GET, '/databases/slow-queries/' . $documents['body']['documents'][0]['$id'], array_merge([
+            'content-type' => 'application/json',
+            'x-appwrite-project' => $this->getProject()['$id'],
+        ], $this->getHeaders()), []);
+        $this->assertEquals(200, $documents['headers']['status-code']);
+        //$this->assertEquals($document['body']['queries'][0], 'sleep("$id", 1)');
+        //$this->assertEquals($document['body']['queries'][0], 'sleep("$id", 1)');
```

Comment:
Let's remove any commented code, if we need it later it's always in git history

Reply:
done!